dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -475,7 +478,7 @@ object LagomPlugin extends AutoPlugin {
 
   private lazy val startKafkaServerTask = Def.task {
     val log = new SbtLoggerProxy(state.value.log)
-    val zooKeeperPort = lagomKafkaZookeperPort.value
+    val zooKeeperPort = lagomKafkaZookeeperPort.value.getOrElse(lagomKafkaZookeperPort.value)
```

Comment:
Another concern is that we'll be stuck with an `Option` type for the new setting forever.

I'm not that expert with sbt, but would it work to do this?

```scala
val lagomKafkaZookeeperPort = settingKey[Int]
// ...
lagomKafkaZookeeperPort := lagomKafkaZookeperPort.value
```

I think that will correctly resolve any overridden value in the build.